src/actions/task-definitions.js

```diff
@@ -92,14 +92,13 @@ tasks.newNotebook = new ExternalLinkTask({
   url: "/new"
 });
 
+// this overrides the browser default's ctrl+s but otherwise does nothing.
 tasks.saveNotebook = new UserTask({
   title: "Save Notebook",
   keybindings: ["ctrl+s", "meta+s"],
   displayKeybinding: `${commandKey}+s`,
   preventDefaultKeybinding: true,
-  callback() {
-    dispatcher.saveNotebookToServer();
-  }
+  callback() {}
```

Comment:
Glitch pops up a little dialog saying "pssst, glitch autosaves!" when you use this shortcut. Might be something to think about for the future.

Reply:
coincidentally, I brought up this idea w/ @bcolloran yesterday afternoon! Agree we could account for that.

Reply:
Filed #1586